lib/ecto/migration.ex

```diff
@@ -457,7 +459,7 @@ defmodule Ecto.Migration do
   end
 
   @doc """
-  Returns a table struct that can be given on create, alter, etc.
+  Returns a table struct that can be given to create, alter, etc.
```

Comment:
What do you think about?

> Returns a table struct that can be given to `create/2`, `alter/2`, `drop/1`, etc.

Reply:
Yep, I think linking the functions is a good idea.